src/install/unix.rst

```diff
@@ -23,6 +23,10 @@ release are the canonical sources of installation information. However, many
 systems have gotchas that you need to be aware of. In addition, dependencies
 frequently change as distributions update their archives.
 
+**Hint**: It worth considering to use LXC-, Docker-conteiners, AMI images or
```

Comment:
It is worth considering using packaging tools such as LXC, Docker containers, AMI images, or similar solutions to configure CouchDB once, and to use this configuration consistently across all nodes in a cluster.